receiver/elasticapmreceiver/internal/mappers/IntakeV2ToDerivedFields.go

```diff
@@ -0,0 +1,86 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This is a nit, but this mapper's filenames don't really match the Go convention for file naming, which is snake case.